docs/user/metacat/source/authinterface.rst

```diff
@@ -1,112 +1,19 @@
 Metacat Authentication Mechanism
 ================================
-Metacat supports either an internal password file authentication or the use of LDAP 
-as an external authentication mechanism.  It does this by supplying two classes 
-(``AuthFile`` or ``AuthLDAP``) that implement authentication via a password file or 
-an external LDAP server. You may choose the authentication mechanism during initial configuration.
 
-If neither of these choices is suitable for your deployment, a custom authentication mechanism can be built.
-Metacat is written such that this Authentication provider is replaceable with 
-another class that implements the same interface (``AuthInterface``). As 
-an Administrator, you have the choice to provide an alternative implementation 
-of ``AuthInterface`` and then configuring **metacat-site.properties** to use that
-class for authentication instead of LDAP or the internal password file.
+Metacat only supports ORCID authentication as an authentication mechanism. File-based or LDAP
```

Comment:
suggested rewording:
~~Metacat only supports ORCID authentication as an authentication mechanism.~~
Metacat only supports ORCID authentication for administrative users.

Reply:
Is that true? I think Metacat supports the DataONE cn_portal authentication, which, while it currently only has ORCID configured, in fact can be configured to support other Identity Providers such as CILogon (and which we have supported in the past).  How about:
Metacat supports the DataONE Authentication service, which typically delegates to ORCID for user identification.

Reply:
As of the current state of `develop`, the admin config page requires one or more ORCIDs - input is validated as such, and there's no option to configure any other Identity Providers. To do so would also require code changes.

Reply:
Thank you! I have updated the wording.